src/phone-input/country-select-dropdown.js

```diff
@@ -0,0 +1,106 @@
+/*
+Copyright (c) 2018-2019 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+
+import React from 'react';
+import {List, AutoSizer} from 'react-virtualized';
+
+import Flag from './flag.js';
+import {SIZE} from './constants.js';
+import {
+  StyledCountrySelectDropdownContainer as DefaultContainer,
+  StyledCountrySelectDropdownListItem as DefaultListItem,
+  StyledCountrySelectDropdownFlagColumn as DefaultFlagColumn,
+  StyledCountrySelectDropdownNameColumn as DefaultNameColumn,
+  StyledCountrySelectDropdownDialcodeColumn as DefaultDialcodeColumn,
+} from './styled-components.js';
+import {getOverrides} from '../helpers/overrides.js';
+
+import type {CountrySelectDropdownPropsT} from './types.js';
+
+export default function CountrySelectDropdown(
+  props: CountrySelectDropdownPropsT,
+) {
+  const {
+    children,
+    dropdownHeight = '400px',
```

Comment:
nit: may worth to move this into a constants file? like here https://github.com/uber-web/baseui/blob/master/src/button/constants.js

Reply:
ended up pulling these out into constants because they were used in a few places